src/main.rs

```diff
@@ -120,9 +123,24 @@ fn status_endpoint(_: Request, res: Response<Fresh>) {
         Err(_) => None,
     };
 
-    let response_body = build_response_json(people_present, raspi_temperature);
-    res.write_all(response_body.as_bytes()).unwrap();
-    res.end().unwrap();
+    // Get response body
+    let body = build_response_json(people_present, raspi_temperature);
+    let body_bytes = body.as_bytes();
+
+    // Set headers
+    {
+        let mut headers = res.headers_mut();
+        headers.set(header::ContentLength(body_bytes.len() as u64));
+        headers.set(header::ContentType("application/json; charset=utf-8".parse().unwrap()));
+        headers.set(header::CacheControl(vec![header::CacheDirective::NoCache]));
+        headers.set(header::AccessControlAllowOrigin::Any);
+    }
```

Comment:
Exactly, if I don't do that the borrow isn't returned on line 140.

Should I add a comment describing that?